zookeeper-server/src/main/java/org/apache/zookeeper/backup/BackupUtil.java

```diff
@@ -0,0 +1,76 @@
+package org.apache.zookeeper.backup;
```

Comment:
util package?

Reply:
We want to keep this here to minimize the difference from the OSS patch.